app/controllers/pseuds_controller.rb

```diff
@@ -104,8 +106,14 @@ def create
   # PUT /pseuds/1.xml
   def update
     @pseud = @user.pseuds.find_by(name: params[:id])
+    authorize @pseud if logged_in_as_admin?
     default = @user.default_pseud
     if @pseud.update(pseud_params)
+      if logged_in_as_admin? && @pseud.ticket_url.present?
```

Comment:
Do we not log the admin's activity if there's no ticket? (I'm out of the loop and have a migraine, so I might be missing a lot of context.)

Reply:
We had to introduce the `ticket_url.present?` check for profiles (#4218) due to [an edge case](https://otwarchive.atlassian.net/browse/AO3-6156?focusedCommentId=358635) where admin activity was logged for empty edits. The same edge case happens here -- without `@pseud.ticket_url.present?`, `RAILS_ENV=test bundle exec cucumber features/other_a/pseuds.feature:177` will fail.

Reply:
We can change the behavior of both in the future if PAC wants it logged. For now, let's keep them consistent and log nothing.